### PR TITLE
cpp: Add all comparison operators for address and bytes32

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by chfast** [comment]
At this point, I think we should add all other comparison operators.

**Comment by axic** [comment]
@chfast is this what you meant?

Need to squash them before merging. And need to include them in the unit tests. @chfast do you want to take this over?

**Comment by chfast** [comment]
> @chfast is this what you meant?
> 
> Need to squash them before merging. And need to include them in the unit tests. @chfast do you want to take this over?

Yes. And yes, I will finish this.

File: include/evmc/evmc.hpp [hunk 1]
```diff
@@ -50,7 +50,6 @@ using uint256be = bytes32;
 /// Loads 64 bits / 8 bytes of data from the given @p bytes array in big-endian order.
 constexpr inline uint64_t load64be(const uint8_t* bytes) noexcept
 {
-    // TODO: Report bug in clang incorrectly optimizing this with AVX2 enabled.
```
Comment (chfast) [inline include/evmc/evmc.hpp]: Reported and fixed: https://bugs.llvm.org/show_bug.cgi?id=42708

**Comment by axic** [comment]
Looks good to me!

**Pull request merged**